cromwell-logging/src/test/scala/cromwell/logging/BusinessLoggingActorSpec.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.logging
```

Comment:
We do it by hand - partially because no one wanted to put one together and partially because there's a lot of subjectivity in scala. Personally I don't like those things because I feel people overly rely on the automated checking when a lot of things aren't easily codified.

We do have a (https://gist.github.com/geoffjentry/17e8baeb03c2678a10fa)[style guide] - which I just realized wasn't publicly listed on the cromwell site. You'll note we're not 100% in compliance ourselves, but we try.